.idea/codeStyles/Project.xml

```diff
@@ -12,6 +12,13 @@
       <option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="50" />
       <option name="JD_ALIGN_PARAM_COMMENTS" value="false" />
     </JavaCodeStyleSettings>
+    <JetCodeStyleSettings>
```

Comment:
Don't know why it says `Jet` but this is to do the same Kotlin setting as same as Java.